antarest/study/storage/output_service.py

```diff
@@ -649,31 +681,32 @@ def get_output_variables_view(
             st_storage_id,
         )
 
+        # Create a temporary file
+        fh, path = tempfile.mkstemp(dir=self._study_service.config.storage.tmp_dir)
+        os.close(fh)
+        tmp_path = Path(path)
+
         # Calls the aggregation with the right arguments
-        export_format = TableExportFormat.PARQUET
-        download_name = f"aggregated_output_{study_id}_{output_id}{export_format.suffix}"
-        download_log = f"Exporting aggregated output data for study '{study_id}' as {export_format} file"
-        download_id = self.aggregate_output_data(
+        task_id = self.aggregate_output_data(
             study_id,
             output_id,
             output_identifier.query_file,
             frequency,
             TableExportFormat.PARQUET,
             [variable_name],
             [output_identifier.get_id_for_aggregation()],
-            download_name,
-            download_log,
-            DEFAULT_DOWNLOAD_EXPIRATION_TIME,
+            tmp_path,
         )
-        self._file_transfer_manager.get_download_metadata(download_id, True)
-        download = self._file_transfer_manager.fetch_download(download_id)
+
+        # Wait for the aggregation to end
+        self._task_service.await_task(task_id)
 
         # Transform the dataframe to have the expected format
         if cluster_id := output_identifier.get_sub_id_for_aggregation():
-            dataframe = pd.read_parquet(Path(download.path), columns=[MCYEAR_COL, variable_name, CLUSTER_ID_COL])
+            dataframe = pd.read_parquet(tmp_path, columns=[MCYEAR_COL, variable_name, CLUSTER_ID_COL])
             dataframe = dataframe[dataframe[CLUSTER_ID_COL] == cluster_id].drop(columns=[CLUSTER_ID_COL])
         else:
-            dataframe = pd.read_parquet(Path(download.path), columns=[MCYEAR_COL, variable_name])
+            dataframe = pd.read_parquet(tmp_path, columns=[MCYEAR_COL, variable_name])
```

Comment:
we need to make sure to remove the temporary file once we don't need it any more.